eureka2-ext/eureka2-aws/src/main/java/com/netflix/eureka2/ext/aws/S3OverridesConfiguration.java

```diff
@@ -0,0 +1,14 @@
+package com.netflix.eureka2.ext.aws;
+
+import com.netflix.archaius.annotations.DefaultValue;
+
+/**
+ * @author David Liu
+ */
+public interface S3OverridesConfiguration extends AwsConfiguration {
```

Comment:
In this case for the S3 specific use case we need to have some S3 specific properties (bucketName and prefix). It extends AwsConfiguration.